server/api/include/irods/rs_get_resource_info_for_operation.hpp

```diff
@@ -0,0 +1,11 @@
+#ifndef IRODS_RS_GET_RESOURCE_INFO_FOR_OPERATION_HPP
+#define IRODS_RS_GET_RESOURCE_INFO_FOR_OPERATION_HPP
+/// \file
+
+#include "irods/dataObjInpOut.h"
+
+struct RsComm;
+
+int rs_get_resource_info_for_operation(RsComm* _rsComm, DataObjInp* _dataObjInp, char** _out_info);
```

Comment:
We will need to add docs similar or identical to the `rc_*` version, I think...

Reply:
You mean to the `rs_*` version ?  If so, yes, I was anticipating doing so, but was waiting  until people were satisfied with the `rc_*` bits because that'll make for overall less procedure.

Reply:
I see, I parsed your meaning / grammar badly, yes in any case `rc_* -> rs_*` will be the direction I'll copy the docs. Since this involves duplicating a source of truth, as I said, we'll wait til everyone is satisfied with the `rc_* `version

Reply:
TODO entered into this file as placeholder.

Reply:
Sounds good

Reply:
No comments have been coming on the docs themselves, so it seems they're good? waiting for input before resolving....

Reply:
The docs look fine. We can tweak them more later if necessary too.

Please mark as resolved.

Reply:
They now have been, yes.